src/chunk.c

```diff
@@ -1946,3 +1947,10 @@ ts_chunk_drop_chunks(PG_FUNCTION_ARGS)
 
 	PG_RETURN_NULL();
 }
+
+Datum
+ts_chunks_in(PG_FUNCTION_ARGS)
+{
+	ereport(ERROR, (errmsg("chunks_in function must appear in the WHERE clause, needs to be on the top level and can't be combined with ORs")));
```

Comment:
Add error code.
Clarify error like this:

```
ereport(ERROR,
              (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
                errmsg("function \"chunks_in\" cannot be invoked directly"),
                 errhint("The function must appear in a WHERE clause.")));
```
I think the OR stuff can be skipped as that is handled in a separate error. Not sure what "be at the top level" means though, so maybe skip that too?

Reply:
I've added the error code and changed the error message. However I think it's better to provide more helpful information in the error message since hints might be off for many.